components/invitation_modal/index.js

```diff
@@ -50,16 +51,19 @@ export function mapStateToProps(state) {
     const isLicensed = license && license.IsLicensed === 'true';
     const isGroupConstrained = Boolean(currentTeam.group_constrained);
     const canInviteGuests = !isGroupConstrained && isLicensed && guestAccountsEnabled && haveITeamPermission(state, {team: currentTeam.id, permission: Permissions.INVITE_GUEST});
+    const isCloud = license.Cloud === 'true';
+    const freeTierWithNoFreeSeats = isCloud && subscriptionStats && subscriptionStats.is_paid_tier === 'false' && subscriptionStats.remaining_seats <= 0;
```

Comment:
Is it possible subscriptionStats might be empty here? true && {} (if subscriptionStats is empty) would mean that the next case (subscriptionStats.is_paid_tier) access would throw an error?

Reply:
thanks @nickmisasi , good catch. I will validate using the isEmpty lodash function.